default.nix

```diff
@@ -15,34 +15,48 @@ rec {
       # be split out: https://github.com/NixOS/nixpkgs/issues/132340. Once that happens,
       # this hack will no longer be needed (and will no longer work)
       gcc-hack = lib.optionalString env.cc.isGNU ''
-           libcxxflags=$out/nix-support/libcxx-cxxflags
-           libcflags=$out/nix-support/libc-cflags
-           echo -isystem ${env.cc.cc}/include/c++/${env.cc.cc.version} >> $libcxxflags
-           echo -isystem ${env.cc.cc}/include/c++/${env.cc.cc.version}/${env.hostPlatform.config} >> $libcxxflags
-           echo -isystem ${env.cc.cc}/lib/gcc/${env.hostPlatform.config}/${env.cc.cc.version}/include >> $libcflags
+        libcxxflags=$out/nix-support/libcxx-cxxflags
```

Comment:
You need to split linting out into a separate PR

Reply:
will fix